esp-hal/src/uart.rs

```diff
@@ -1356,45 +1425,40 @@ where
         //
         // Ideally this should be configurable once we have a solution for https://github.com/esp-rs/esp-hal/issues/1111
```

Comment:
we don't need to resolve this here and now but at least 1111 is resolved - since we are touching the code already, we could at least update the comment (or introduce a config-option - probably similar effort)